.github/workflows/documentation.yml

```diff
@@ -19,7 +19,7 @@ env:
 jobs:
   # For any push and PR, build the documentation from the ocaml comments
   # If this build fails, the documentation workflow stops
-  # If it succed, an artifact is made with the generated documentation
+  # If it succeed, an artifact is made with the generated documentation
```

Comment:
```suggestion
  # If it succeeds, an artifact is made with the generated documentation
```

Reply:
This is an incremental spelling correction! :D